app/services/LocationService.js

```diff
@@ -80,35 +80,39 @@ export class LocationData {
       }
 
       // Backfill the stationary points, if available
-      // The assumption is that if we see a gap in the data, the
-      // best guess is that the person was in that location for 
-      // the entire time.  This makes it easier for a health authority
+      // The assumption is that if we see a gap in the data, and the 
+      // device hasn't moved significantly, then we can fill in the missing data
+      // with the current location.  This makes it easier for a health authority
       // person to have a set of locations over time, and they can manually
       // redact the time frames that aren't correct.
       if (curated.length >= 1) {
         let lastLocationArray = curated[curated.length - 1];
 
-        // Figure out the time to start the backfill from.  Default is
-        // the time of the last entry in the location set.
-        // To protect ourselves, we won't backfill more than the
-        // maxBackfillTime
-        let lastRecordedTime = lastLocationArray['time'];
-        if ((unixtimeUTC - lastRecordedTime) > this.maxBackfillTime) {
-          lastRecordedTime = unixtimeUTC - this.maxBackfillTime;
-        }
+        var nearby = areLocationsNearby(
```

Comment:
Convention seems to be `isX` for booleans

Reply:
yeah, I know.  Always bothers me when talking about plurals.  I'll change.

Reply:
`areX` would be fine too :)

It's the difference of nearby might be nearby points, vs areNearby is clearly a boolean

Reply:
it's now `isLocationsNearby`